interpreter.go

```diff
@@ -360,6 +360,21 @@ func (i *interpreter) objectIndex(loc *LocationRange, obj value, f string, offse
 	}
 }
 
+func (i *interpreter) evalAdd(leftVal, rightVal value) (value, error) {
+	switch leftVal := leftVal.(type) {
+	case *valueNumber:
+		left := leftVal.value
+		right := rightVal.(*valueNumber).value
+		return makeValueNumber(left + right), nil
+	case *valueString:
+		left := leftVal.value
+		right := rightVal.(*valueString).value
+		return makeValueString(left + right), nil
+	}
+	// TODO(sbarzowski) More types and more graceful error handling
+	panic("unknown type")
```

Comment:
I know you plan to flush this out in the future, but a simple `panic(fmt.Sprintf("unknown type: %v", leftVal))` will probably avoid some potential headscratching.